### ISSUE TITLE
Unable to access webmail account

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2184]
**Reported by pattraco on 2012-03-19 12:47**
Hi I am unable to access this website with NVDA

http://webmail.pattraco.com/

I would like to access this website and my email account on this using NVDA. NVDA simply refuses to move forward. 

Please treat this as urgent and important. I need help !

**Comment by ehollig** [comment]
The website that was referenced [http://webmail.pattraco.com/](http://webmail.pattraco.com/) does not exist anymore.
